src/test/java/POM/HomePage.java

```diff
@@ -0,0 +1,92 @@
+package POM;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+
+public class HomePage extends BasePage {
+    By userAvatarIcon = By.cssSelector("img.avatar");
+    By allSongsLocator = By.xpath("//a[@href='#!/songs']");
+    By firstPlaylist = By.cssSelector(".playlist:nth-child(3)");
+    By playlistInputField = By.xpath("//input[@name='name']");
+    By renamePlaylistInput = By.cssSelector("[data-testid='inline-playlist-name-input'");
+    By myNewPlaylistVerify = By.xpath("//a[text()='My new playlist']");
+    By createPlaylistButton = By.cssSelector("[data-testid='sidebar-create-playlist-btn']");
+    By soundBarLocator = By.cssSelector("[alt='Sound bars']");
+    By selectSimplePlaylist = By.cssSelector("[data-testid='playlist-context-menu-create-simple']");
+    By myNewPlaylistName = By.xpath("//a[text()='My playlist']");
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.


I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```